pingdom/src/main/java/com/pulse/dbcp/DataBaseUtility.java

```diff
@@ -0,0 +1,51 @@
+package com.pulse.dbcp;
+
+import org.apache.commons.dbcp2.BasicDataSource;
+
+public class DataBaseUtility {
+
+
+    private static final String DRIVER_CLASS_NAME = "org.postgresql.Driver";
+    //enter your database name at the end instead of pulse
+    private static final String DB_URL = "jdbc:postgresql://localhost/pulse";
+    //enter your local database username
+    private static final String DB_USER = "db_user";
+    //password for your respective db
+    private static final String DB_PASSWORD = "db_user_password";
+
+    private static final int CONN_POOL_SIZE = 10;
+
+
+    private BasicDataSource bds = new BasicDataSource();
```

Comment:
`Bds`?  u can name this object as `basicDataSource`, there by we can have more context on the object when we read the code

Reply:
ok I'll change it.